alexandria/core/serializers.py

```diff
@@ -100,9 +100,25 @@ def create(self, validated_data):
 
 
 class CategorySerializer(SlugModelSerializer):
+    parent = serializers.ResourceRelatedField(required=False, read_only=True)
+    children = serializers.ResourceRelatedField(
```

Comment:
There is no backend limitation on children depth?

Reply:
No, currently not